--- Publican-v3.0.0/t/110.CreateBrand.t.orig	2012-10-31 03:09:22.000000000 +0100
+++ Publican-v3.0.0/t/110.CreateBrand.t	2012-11-01 13:03:58.576844498 +0100
@@ -3,6 +3,7 @@
 
 use Test::More tests => 5;
 use File::Path;
+use Cwd qw(abs_path);
 
 BEGIN {
     use_ok('Publican');
@@ -21,6 +22,7 @@
         = Publican::CreateBrand->new(
         { name => 'Test_Brand', lang => 'en-US' } );
 };
+$creator->{config}->param('common_content', abs_path("blib/datadir/Common_Content"));
 
 isa_ok( $creator, 'Publican::CreateBrand',
     'creating a Publican::CreateBrand' );
