--- Publican-2.8/datadir/xsl/web-spec.xsl.orig	2011-09-22 04:02:20.000000000 +0200
+++ Publican-2.8/datadir/xsl/web-spec.xsl	2012-10-31 20:48:04.972635095 +0100
@@ -62,6 +62,8 @@
 <xsl:if test="$translation = '1'"><xsl:value-of select="$language"/> translation of <xsl:value-of select="$docname"/>
 
 %description -l <xsl:value-of select="$lang"/>  -n <xsl:value-of select="$book-title"/>-<xsl:value-of select="$lang"/> </xsl:if>
+<xsl:if test="$translation != '1'"><xsl:text>
+</xsl:text></xsl:if>
 <xsl:value-of select="$abstract" />
 
 %prep
